internal/pkg/template/repository/manager/manager.go

```diff
@@ -0,0 +1,65 @@
+package manager
+
+import (
+	"crypto/sha256"
+	"fmt"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
+	"github.com/keboola/keboola-as-code/internal/pkg/git"
+	"github.com/keboola/keboola-as-code/internal/pkg/log"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/template/repository"
+)
+
+type Manager struct {
+	logger       log.Logger
+	repositories map[string]*GitRepository
+}
+
+type GitRepository struct {
+	Dir string
+	Fs  filesystem.Fs
+}
+
+func hashRepositoryName(repo model.TemplateRepository) string {
```

Comment:
- Nie je to hash `name`, ... je to privatna funkcia, kludne iba `hash`
- Je to mozne definovat aj ako metodu nad `model.TemplateRepository.Hash()` (ak tam pridas este type a path, tak to bude univerzalne) (ale aj takto je to OK)

Reply:
Takhle to myslíš? https://github.com/keboola/keboola-as-code/pull/603/commits/abe39ea0866d648608edb43d3d6339d9af34431a#diff-650390af41515501397bae4d7e78ffb03564f379df701d1eae7be0965bbf1c32R26 Že se do toho hashe bude současně dávat path i url, i když může být některý z nich prázdný? (Což teda funkci neohrožuje, jen jestli to dobře chápu 🙂)

Reply:
jj, tak som to myslel